boards/arm/nrf52_pca20020/doc/nrf52_pca20020.rst

```diff
@@ -0,0 +1,354 @@
+| SX1509B  | 0x3e    |
++----------+---------+
+| LPS22HB  | 0x5c    |
++----------+---------+
+| HTS221   | 0x5f    |
++----------+---------+
+| CCS811   | 0x5a    |
++----------+---------+
+
+External I2C Bus
+----------------
+
+The external I2C bus (I2C_1) can be found on the P4 header and the P5 and P7
+connectors.
+
++---------+---------+
+| SOC Pin | Signal  |
++=========+=========+
+| P0.14   | SDA_EXT |
++---------+---------+
+| P0.15   | SCL_EXT |
++---------+---------+
+
+Pin Header
+----------
+
+This is the pinout of the P4 pin header. Some of the SOC GPIO pins and I2C GPIO
+expander pins are accessible through it. It also allows attaching external
+devices to the four on-board N-channel MOSFET transistors.
+
++-----+---------------+-----------------------+
+| Pin | Device        | Signal / Device Pin   |
++=====+===============+=======================+
+| 1   | SOC           | SCL_EXT / P0.15       |
++-----+---------------+-----------------------+
+| 2   | SOC           | SDA_EXT / P0.14       |
++-----+---------------+-----------------------+
+| 3   | SOC           | ANA/DIG0 / P0.02      |
++-----+---------------+-----------------------+
+| 4   | SOC           | ANA/DIG1 / P0.03      |
++-----+---------------+-----------------------+
+| 5   | SOC           | ANA/DIG2 / P0.04      |
++-----+---------------+-----------------------+
+| 6   |               | GND                   |
++-----+---------------+-----------------------+
+| 7   | GPIO Expander | Pin 0                 |
++-----+---------------+-----------------------+
+| 8   | GPIO Expander | Pin 1                 |
++-----+---------------+-----------------------+
+| 9   | GPIO Expander | Pin 2                 |
++-----+---------------+-----------------------+
+| 10  | GPIO Expander | Pin 3                 |
++-----+---------------+-----------------------+
+| 11  | MOSFET 1      | Drain                 |
++-----+---------------+-----------------------+
+| 12  | MOSFET 1      | Source                |
++-----+---------------+-----------------------+
+| 13  | MOSFET 2      | Drain                 |
++-----+---------------+-----------------------+
+| 14  | MOSFET 2      | Source                |
++-----+---------------+-----------------------+
+| 15  | MOSFET 3      | Drain                 |
++-----+---------------+-----------------------+
+| 16  | MOSFET 3      | Source                |
++-----+---------------+-----------------------+
+| 17  | MOSFET 4      | Drain                 |
++-----+---------------+-----------------------+
+| 18  | MOSFET 4      | Source                |
++-----+---------------+-----------------------+
+| 19  |               | VDD                   |
++-----+---------------+-----------------------+
+| 20  |               | GND                   |
++-----+---------------+-----------------------+
+
+MOSFETs
+~~~~~~~
+
+The MOSFETs are attached to the following SOC GPIO pins:
+
++----------+----------+
+| Device   | Gate Pin |
++----------+----------+
```

Comment:
Use ``====``'s as the row separator here to indicate the previous line is a heading.